pkgs/development/compilers/go/1.11.nix

```diff
@@ -56,6 +57,10 @@ stdenv.mkDerivation rec {
     substituteInPlace misc/cgo/testcarchive/carchive_test.go \
       --replace '#!/usr/bin/env bash' '#!${stdenv.shell}'
 
+    # Patch the mimetype database location which is missing on NixOS.
+    substituteInPlace src/mime/type_unix.go \
+      --replace '/etc/mime.types' '${mailcap}/etc/mime.types'
```

Comment:
Use the package `mime-types` instead

Reply:
According to #50987 mime-types is outdated, so I used this instead.

Reply:
Ah I see, didn't know that